esp/applications/sql2ecl/esp.yaml

```diff
@@ -0,0 +1,27 @@
+esp:
+   instance: sql2ecl
+   description: SQL2ECL Service
+   daliServers: dali
+   auth: ldap
+   tls: true
+   port: 8880
+   enableSEHMapping: true
+   httpConfigAccess: true
+   logLevel: 1
+   maxBacklogQueueSize: 200
+   portalurl: http://hpccsystems.com/download
+   logDir: "-"
```

Comment:
directories.yaml is only there to make sure I don't break anything in ESP that relies on it, I plan to remove it at some point.  This logDir directory is easy to set from command line, config, or helm.  It can change but currently makes it easy to log to a particular file.  Default is simply stderr (or whichever std stream components log to by default).